[16.0] [MIG] stock_financial_risk

olveracuenca opened the pull request.

nbessi committed:
[ADD] credit control dunning fees module

nbessi committed:
[ADD] docstring

nbessi committed:
[FIX] FeesComputer model specification to act as a abstract class without needing overriding

nbessi committed:
[IMP] fees column in report and mail

nbessi committed:
[FIX] dunning fees computer to register itself

nbessi committed:
[RM] unused import

nbessi committed:
[PEP8]

nbessi committed:
[IMP] module documentation

nbessi committed:
[ADD] dunniing fees translations

nbessi committed:
[FIX] name of level in tests

nbessi committed:
[TYPOS]

nbessi committed:
Set all addons to uninstallable

nbessi committed:
Rename all addons to xxx_unported to be travis compliant

Commit:
Revert "Rename all addons to xxx_unported to be travis compliant"

This reverts commit 7eb182744f3057dd5cb9e35faf230412aa7a2331.

Commit:
[FIX] move all unported addons into __unported__ folder instead of adding _unported suffix

nbessi committed:
[FIX] flake8 PEP8 for module account_credit_control_dunning_fees

Commit:
[FIX] flake8 PEP8 for module account_credit_control_dunning_fees

Commit:
[FIX] pep8 line length to 80, activate pep8 check in travis, and add noqa on scenario while waiting improvement of quality-tools

nbessi committed:
Add missing security on `credit.control.dunning.fees.computer`

Commit:
Move account_credit_control_dunning_fees to root folder

Commit:
Extract the module description to README.rst

Commit:
Port Dunning fees to the new API, activate the tests for 8.0

Commit:
Add fees in the report

Commit:
Use the new API instead of mocks for FixedFeesTester

It will be easier to write additional tests

Commit:
Remove required on the currency and fallback to company's currency...

... on the policy level if it is empty.

Also, it fixes a bug when a credit line had no currency and the level
had the company's currency. As False is different than any currency,
it entered in res_currency.compute() with a False currency. It must
just fallback on the company's currency if the credit line's currency
is empty.

Commit:
Remove the unused mako template

Commit:
Include the fees in the open amount

Display the fees column before the open amount, it is more logical since it is
invoice amount + fees amount.

Commit:
Update translation of account_credit_control_dunning_fees

rschnapka committed:
german translations upto invoice currency

Commit:
Remove post_install and at_install instructions on tests

With that, the suite is run even when we update another module and that not
what we expect.

Commit:
Update translations, translate words of the report in German

gurneyalex committed:
Add OCA as author of OCA addons

In order to get visibility on https://www.odoo.com/apps the OCA board has
decided to add the OCA as author of all the addons maintained as part of the
association.

Commit:
[IMP] Include the fees in the total due

yvaucher committed:
Add missing default oca icons

oca-transbot committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
[UPD] prefix versions with 8.0

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
OCA Transbot updated translations from Transifex

sbidoul committed:
[FIX] remove en.po that was erroneously created by transbot

pedrobaeza committed:
[MIG] Rename manifest files

carlosdauden committed:
[ADD] partner_financial_risk (#292)

carlosdauden committed:
[9.0][ADD] partner_sale_risk: New module

carlosdauden committed:
[9.0][FIX] partner_financial_risk: Refund were not contemplated

grindtildeath committed:
[MIG] account_credit_control_dunning_fees: Migration to 10.0

carlosdauden committed:
[9.0][IMP] partner_financial_risk: Improve performance (#385)

* [9.0][IMP] partner_financial_risk: Improve performance

* [9.0][IMP] partner_financial_risk: Improve tests and partner hierarchy

* [9.0][IMP] partner_stock_risk: Test

carlosdauden committed:
[9.0] [ADD] partner_stock_risk (#300)

* [9.0][ADD] partner_stock_risk: New module

* [9.0][WIP] partner_stock_risk: New module

* [9.0][WIP] partner_stock_risk: Improve code

* [9.0][FIX] partner_stock_risk: Fix test

* [9.0][IMP] partner_stock_risk: Translate

carlosdauden committed:
[9.0][IMP] partner_stock_risk: Improve tests

oca-transbot committed:
OCA Transbot updated translations from Transifex

Ehtaga committed:
[MIG] Partner Financial Risk : Migration to 10.0

Ehtaga committed:
[MIG] Partner Sale Risk : Migration to 10.0

chienandalu committed:
[MIG] partner_stock_risk: Migration to 10.0

carlosdauden committed:
[9.0][FIX] partner_financial_risk: Error when risk_total includes draft_invoices

chienandalu committed:
[IMP] Minor changes

chienandalu committed:
[9.0][IMP] partner_financial_risk: Add track visibility to credit limit

carlosdauden committed:
[FIX] partner_financial_risk: Invoice refunds add risk

- Refunds adds partner risk instead of reduce.
- If date_maturity changes to previous today date "never" compute partner risk.

carlosdauden committed:
[IMP] partner_financial_risk: New fields in view. Improve code.

carlosdauden committed:
[10.0][IMP] partner_financial_risk: Improve multicompany cron (#530)

* [9.0][IMP] partner_financial_risk: Improve multicompany cron

* [9.0][IMP] partner_financial_risk: Exec with sudo

(cherry picked from commit 6590bbf)

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-transbot committed:
OCA Transbot updated translations from Transifex

carlosdauden committed:
[MIG] partner_financial_risk: Migration to 11.0

pedrobaeza committed:
[REN] partner_financial_risk > account_financial_risk

carlosdauden committed:
[MIG] partner_sale_risk: Migration to 11.0

carlosdauden committed:
[REN] partner_sale_risk > sale_financial_risk

pedrobaeza committed:
[FIX] account_credit_control_dunning_fees: Change currency by SQL

To avoid constraint.

sbidoul committed:
remove obsolete .pot files [ci skip]

oca-travis committed:
[UPD] Update partner_stock_risk.pot

carlosdauden committed:
[IMP] sale_financial_risk: Use new core fields amt_to_invoice and amt_invoiced

carlosdauden committed:
[IMP] account_financial_risk: Improve table layout

carlosdauden committed:
[IMP] sale_financial_risk: Improve layout view. Readme new template

carlosdauden committed:
[IMP] sale_financial_risk: Rename module in translations files

oca-travis committed:
[UPD] Update account_credit_control_dunning_fees.pot

carlosdauden committed:
[IMP] account_financial_risk: Only block normal customer invoices validation

alvarorib committed:
Translated using Weblate (Portuguese (Portugal))

Currently translated at 100.0% (20 of 20 strings)

Translation: account-financial-tools-10.0/account-financial-tools-10.0-account_credit_control_dunning_fees
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-10-0/account-financial-tools-10-0-account_credit_control_dunning_fees/pt_PT/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 95.0% (19 of 20 strings)

Translation: account-financial-tools-10.0/account-financial-tools-10.0-account_credit_control_dunning_fees
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-10-0/account-financial-tools-10-0-account_credit_control_dunning_fees/pt/

ernestotejeda committed:
[MIG] account_financial_risk: Migration to 12.0

ernestotejeda committed:
[MIG] sale_financial_risk: Migration to 12.0

ernestotejeda committed:
[MIG] stock_financial_risk: Migration to 12.0 and renaming

[OCA/partner-contact] partner_stock_risk > stock_financial_risk
and Migration to 12.0

ernestotejeda committed:
[MIG] sale_financial_risk: Add unit tests

carlosdauden committed:
[IMP] sale_financial_risk: Simplify code (Remove no directly related invoices)
Company currency

carlosdauden committed:
[IMP] account_financial_risk: Manage risk in multiple invoice confirmation wizard

OCA-git-bot committed:
[ADD] icon.png

oca-travis committed:
[UPD] Update stock_financial_risk.pot

TDu committed:
[MIG] account_credit_control_dunning_fees: Migration to 12.0

TDu committed:
fixup! [MIG] account_credit_control_dunning_fees: Migration to 12.0

chafique-delli committed:
Translated using Weblate (French)

Currently translated at 7.7% (1 of 13 strings)

Translation: credit-control-12.0/credit-control-12.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-sale_financial_risk/fr/

chafique-delli committed:
Translated using Weblate (French)

Currently translated at 71.2% (52 of 73 strings)

Translation: credit-control-12.0/credit-control-12.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-account_financial_risk/fr/

OCA-git-bot committed:
[UPD] README.rst

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 7.7% (1 of 13 strings)

Translation: credit-control-12.0/credit-control-12.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-sale_financial_risk/pt_BR/

Commit:
Translated using Weblate (Italian)

Currently translated at 79.5% (58 of 73 strings)

Translation: credit-control-12.0/credit-control-12.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-account_financial_risk/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 46.2% (6 of 13 strings)

Translation: credit-control-12.0/credit-control-12.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-sale_financial_risk/it/

jesusVMayor committed:
[FIX]account_financial_risk: risk_exception field explicitly setted to False.

sergio-teruel committed:
[IMP] sale_financial_risk: Fix test crash with other modules due to pricelist

Commit:
[FIX] iterate on recordset

Commit:
Translated using Weblate (Spanish (Chile))

Currently translated at 13.7% (10 of 73 strings)

Translation: credit-control-12.0/credit-control-12.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-account_financial_risk/es_CL/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (73 of 73 strings)

Translation: credit-control-12.0/credit-control-12.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-account_financial_risk/es/

alan196 committed:
[IMP] stock_financial_risk: black, isort

alan196 committed:
[MIG] stock_financial_risk: Migration to V13

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 76.7% (56 of 73 strings)

Translation: credit-control-12.0/credit-control-12.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-account_financial_risk/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (13 of 13 strings)

Translation: credit-control-12.0/credit-control-12.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-sale_financial_risk/pt_BR/

carlosdauden committed:
[IMP] account_financial_risk: Avoid compute all partners when save config settings

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (73 of 73 strings)

Translation: credit-control-12.0/credit-control-12.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-account_financial_risk/pt/

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 23.1% (3 of 13 strings)

Translation: credit-control-12.0/credit-control-12.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-12-0/credit-control-12-0-sale_financial_risk/pt/

carlosdauden committed:
[IMP] account_financial_risk: Add allow overrisk invoice validation option. Improve views and translations.

carlosdauden committed:
[IMP] sale_financial_risk: Bypass risk when payment transaction is authorized

carlosdauden committed:
[IMP] account_financial_risk account_payment_return_financial_risk sale_financial_risk:
 - Optimize code and change stored computed fields to not store to improve multi-company support
 - Improve multi-currency support
 - Convert risk amount fields to clickable link that shows traceability of amount origin
 - New pivot views to risk amount traceability
 - Simplify class style applied on risk fields
 - Migration script to remove old stored computed fields
 - Improve tests to cover new functionallity
 - Update translation files

account_financial_risk:
 - Don't block refund invoice validation when partner has risk exception
 - Allow search partners by risk exception field
 - Remove obsolete cron

sale_financial_risk
 - Create related store commercial_partner_id field in sale order line to simplify computation
 - Rename amt_to_invoice field to risk_amount in sale order line
 - Hook and migration scripts to reduce new fields computing time

TT23765

carlosdauden committed:
[IMP] account_financial_risk account_payment_return_financial_risk sale_financial_risk:
 - Optimize code and change stored computed fields to not store to improve multi-company support
 - Improve multi-currency support
 - Convert risk amount fields to clickable link that shows traceability of amount origin
 - New pivot views to risk amount traceability
 - Simplify class style applied on risk fields
 - Migration script to remove old stored computed fields
 - Improve tests to cover new functionallity
 - Update translation files

account_financial_risk:
 - Don't block refund invoice validation when partner has risk exception
 - Allow search partners by risk exception field
 - Remove obsolete cron

sale_financial_risk
 - Create related store commercial_partner_id field in sale order line to simplify computation
 - Rename amt_to_invoice field to risk_amount in sale order line
 - Hook and migration scripts to reduce new fields computing time

TT23765

MiquelRForgeFlow committed:
[IMP] account_financial_risk: black, isort, prettier

MiquelRForgeFlow committed:
[MIG] account_financial_risk: Migration to 13.0

MiquelRForgeFlow committed:
[IMP] sale_financial_risk: black, isort, prettier

MiquelRForgeFlow committed:
[MIG] sale_financial_risk: Migration to 13.0

MiquelRForgeFlow committed:
[IMP] Add selection: choose which currency should be used in risk

MiquelRForgeFlow committed:
[IMP] Add selection: choose which currency should be used in risk

jesusVMayor committed:
[IMP]sale_financial_risk: Add sale orders in done state to risk calculation if auto done is configured.

leemannd committed:
Translated using Weblate (French)

Currently translated at 11.8% (2 of 17 strings)

Translation: credit-control-13.0/credit-control-13.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-13-0/credit-control-13-0-sale_financial_risk/fr/

sergio-teruel committed:
[IMP] sale_financial_risk: Compute risk for partner invoice. Add inheritable method.

oca-travis committed:
[UPD] Update stock_financial_risk.pot

OCA-git-bot committed:
[UPD] README.rst

carlosdauden committed:
[IMP] sale_financial_risk: Change related risk partner target and rename field

AaronHForgeFlow committed:
[IMP] account_financial_risk: Add specific overpassing group

This commit adds an specific group for overpassing partner credit limit exceptions
In some situations, not only account managers shoud be allowed to overpass exception, depending on the company policies. Merging this code will not affect current instances, as long as account managers will be added to the new group.

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-13.0/credit-control-13.0-stock_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-13-0/credit-control-13-0-stock_financial_risk/

psugne committed:
[MIG] account_financial_risk: Migration to 14.0

psugne committed:
[MIG] sale_financial_risk: Migration to 14.0

psugne committed:
[MIG] stock_financial_risk: Migration to 14.0

Commit:
[FIX] sale_financial_risk: Expected singleton: sale.order()

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 82.3% (14 of 17 strings)

Translation: credit-control-14.0/credit-control-14.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-sale_financial_risk/fr/

Yvesldff committed:
Translated using Weblate (French)

Currently translated at 86.2% (69 of 80 strings)

Translation: credit-control-14.0/credit-control-14.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-account_financial_risk/fr/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 16.2% (13 of 80 strings)

Translation: credit-control-14.0/credit-control-14.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-account_financial_risk/nl/

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 11.7% (2 of 17 strings)

Translation: credit-control-14.0/credit-control-14.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-sale_financial_risk/nl/

carlosdauden committed:
[FIX] sale_financial_risk: Commercial partner not set when value is null

pedrobaeza committed:
[FIX] account_financial_risk: Fix tests fixing a date

There's a part of the risk computation that depends on current date,
and this makes the test to fail when time has past.

We force the maturity date on the invoice we are testing for always
having the field properly computed. An alternative would be to use
``freezegun``, but that means more changes in the test suite.

Commit:
[FIX] sale_financial_risk: Don't use deprecated method

`account.move` `post()` is deprecated since Odoo 14.0.

carlosdauden committed:
[FIX] account_financial_risk: Show ValidationError when post multiple invoices from wizard. Avoid singleton with multi-company _autopost_draft_entries.
 When we post from validate.account.move wizard, validate_move returns 'ir.actions.act_window_close'.
 Fixed issue #147
 TT30969

pedrobaeza committed:
[FIX] stock_financial_risk: Propagate cancel_backorder

Respect inheritance.

oca-travis committed:
[UPD] Update stock_financial_risk.pot

OCA-git-bot committed:
[UPD] README.rst

JoanMForgeFlow committed:
[IMP] account_financial_risk: black, isort, prettier

JoanMForgeFlow committed:
[15.0][MIG] account_financial_risk:Migration to 15.0

JoanMForgeFlow committed:
[FIX]-account-financial-risk: fixed test

JoanMForgeFlow committed:
[IMP] sale_financial_risk: black, isort, prettier

JoanMForgeFlow committed:
[15.0][MIG] sale_financial_risk: Migration to 15.0

MiquelRForgeFlow committed:
[FIX] account_financial_risk: consider include in risk_exception

MiquelRForgeFlow committed:
[FIX] account_financial_risk: consider include in risk_exception

MiquelRForgeFlow committed:
[IMP] account_financial_risk: add risk_amount_exceeded

MiquelRForgeFlow committed:
[FIX] account_financial_risk (bad migration): action_post -> post

carlosdauden committed:
[FIX] account_financial_risk: Remove customer_rank condition to show financial risk info in partner

TT33665

pedrobaeza committed:
[FIX] account_financial_risk: Don't store company_currency_id

The field is not populated on proper time (only once saved) when
recomputing dynamic lines if it's stored, provoking an error.

Moreover, making it stored seems a random decision, as it's not
benefitting the computation of the rest of the fields, as they are not
stored as well.

Being on it, the depends of the other computed field has been adjusted
to the proper field. Needing the other depend to work was a symptom of
this issue that was ignored.

TT33885

pedrobaeza committed:
[FIX] sale_financial_risk: Make tests to work

The import failure was not detected as error in CI, and restoring it,
reveals other problems in the tests.

fredzamoabg committed:
[FIX] account_financial_risk: inherit action_post instead of _post

carlosdauden committed:
[ADD] sale_financial_risk_info: New module.
Extends account financial risk to add consumption info in sales orders.

carlosdauden committed:
[IMP] sale_financial_risk_info: Add pattern options to customize info

ilyasProgrammer committed:
[MIG] sale_financial_risk_info: Migration to 14.0

francesco-ooops committed:
Added translation using Weblate (Italian)

ofonseca-pyming committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: credit-control-14.0/credit-control-14.0-stock_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-stock_financial_risk/es/

enriquemartin committed:
[14.0][IMP] sale_financial_risk_info: apply currency format

ClementChaumienneC2C committed:
Translated using Weblate (French)

Currently translated at 92.5% (74 of 80 strings)

Translation: credit-control-15.0/credit-control-15.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-account_financial_risk/fr/

ClementChaumienneC2C committed:
Translated using Weblate (French)

Currently translated at 85.0% (17 of 20 strings)

Translation: credit-control-15.0/credit-control-15.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-sale_financial_risk/fr/

nimarosa committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (4 of 4 strings)

Translation: credit-control-14.0/credit-control-14.0-stock_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-stock_financial_risk/es_AR/

Commit:
[14.0][IMP] sale_financial_risk: Absolute and percentage values of remaining risk added to partner and Sale Order as computed fields to make it compatible with domain widget.

Forward-port https://github.com/OCA/credit-control/pull/207

Commit:
[14.0][IMP] sale_financial_risk: Absolute and percentage values of remaining risk added to partner and Sale Order as computed fields to make it compatible with domain widget.

LoisRForgeFlow committed:
[FIX] account_financial_risk: issue with currenty rates.

When doing conversions of currencies we cannot ignore the
invoice dates, otherwise differences between de residual currency
amount of the invoices and the computed risk amounts can arise.

Steps to reproduce:

1. Create 2 rates for a secondary currency.
2. Enter an invoice in that secondary currency and set the invoice
   date in the past so it uses the oldest rate.
3. Check financial risk fields.

The fix implemented is a compromise. On one hand, having the most
robust fix would imply iterate all invoices individually. On the
other hand, loosing the performance benefits given by the `read_group`
is also not desired.

carlosdauden committed:
[IMP] account_financial_risk: Alternative to check if move post is called from validate.account.move wizard
We can not return "partner.risk.exceeded.wiz" because validate_move method always returns "{'type': 'ir.actions.act_window_close'}"

ClementChaumienneC2C committed:
Translated using Weblate (French)

Currently translated at 97.5% (81 of 83 strings)

Translation: credit-control-15.0/credit-control-15.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-account_financial_risk/fr/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: credit-control-14.0/credit-control-14.0-sale_financial_risk_info
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-sale_financial_risk_info/it/

Commit:
[15.0][IMP] account_financial_risk: credit limit last modification date field.

Commit:
[15.0][FIX] account_financial_risk: date_credit_limit view in edition mode.

AaronHForgeFlow committed:
[FIX] account_financial_risk: Access error when displaying the partner risk exceeded wizard

Commit:
[MIG] sale_financial_risk_info: Migration to 15.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-15.0/credit-control-15.0-sale_financial_risk_info
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-sale_financial_risk_info/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 50.0% (2 of 4 strings)

Translation: credit-control-14.0/credit-control-14.0-stock_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-14-0/credit-control-14-0-stock_financial_risk/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 25.0% (5 of 20 strings)

Translation: credit-control-15.0/credit-control-15.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-sale_financial_risk/it/

nguyenminhchien committed:
[IMP] account_credit_control_dunning_fees: black, isort, prettier

nguyenminhchien committed:
[16.0][MIG] account_credit_control_dunning_fees: Migration to 16.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 53.8% (49 of 91 strings)

Translation: credit-control-15.0/credit-control-15.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-account_financial_risk/it/

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 36.2% (33 of 91 strings)

Translation: credit-control-15.0/credit-control-15.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-account_financial_risk/sl/

Urvisha-OSI committed:
[IMP] stock_financial_risk: pre-commit stuff

Urvisha-OSI committed:
[MIG] stock_financial_risk: Migration to 16.0

mymage committed:
Translated using Weblate (Italian)

Currently translated at 53.8% (49 of 91 strings)

Translation: credit-control-15.0/credit-control-15.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-15-0/credit-control-15-0-account_financial_risk/it/

stefan-tecnativa committed:
[IMP] sale_financial_risk: pre-commit stuff

stefan-tecnativa committed:
[MIG] sale_financial_risk: Migration to 16.0

andreampiovesana committed:
Update __manifest__.py

"application": False,

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/

carlosdauden committed:
[FIX] xxx_financial_risk: Inconsistent 'compute_sudo' for several computed fields

sanrav committed:
Translated using Weblate (French)

Currently translated at 74.7% (210 of 281 strings)

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/fr/

astirpe committed:
[FIX] account_credit_control: avoid AccessError

OCA-git-bot committed:
Merge PR #287 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_credit_control 16.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

mymage committed:
Translated using Weblate (Italian)

Currently translated at 6.7% (19 of 281 strings)

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 6.7% (19 of 281 strings)

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 6.7% (19 of 281 strings)

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/it/

OCA-git-bot committed:
Merge PR #285 into 16.0

Signed-off-by sergiocorato

victoralmau committed:
[IMP] account_credit_control: Add Payment Reference from invoice to communication table

TT41337

victoralmau committed:
[FIX] account_credit_control: Add table to email content

ernesto-garcia-tecnativa committed:
[IMP] account_financial_risk: black, isort, prettier

OCA-git-bot committed:
Merge PR #291 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update account_credit_control.pot

OCA-git-bot committed:
account_credit_control 16.0.1.0.2

OCA-git-bot committed:
[UPD] addons table in README.md

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/

ernesto-garcia-tecnativa committed:
[MIG] account_financial_risk: Migration to 16.0

OCA-git-bot committed:
Merge PR #292 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update account_financial_risk.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (282 of 282 strings)

Translation: credit-control-16.0/credit-control-16.0-account_credit_control
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control/es/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_financial_risk/

ernesto-garcia-tecnativa committed:
[IMP] sale_financial_risk_info: black, isort, prettier

ernesto-garcia-tecnativa committed:
[MIG] sale_financial_risk_info: Migration to 16.0

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (91 of 91 strings)

Translation: credit-control-16.0/credit-control-16.0-account_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_financial_risk/es/

EduardoJarsa commented:
@alan196 can you review this PR?

alan196 commented:
There is already a PR in OCA https://github.com/OCA/credit-control/pull/276

Please check the last commit, this module may have the same issue

OCA-git-bot committed:
Merge PR #294 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_financial_risk_info.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-sale_financial_risk_info
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-sale_financial_risk_info/

ernesto-garcia-tecnativa committed:
[IMP] account_financial_risk: problems of partner view are fixed

OCA-git-bot committed:
Merge PR #295 into 16.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
account_financial_risk 16.0.1.1.0

OCA-git-bot committed:
[UPD] addons table in README.md

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: credit-control-16.0/credit-control-16.0-sale_financial_risk_info
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-sale_financial_risk_info/es/

OCA-git-bot committed:
Merge PR #274 into 16.0

Signed-off-by simahawk

Commit:
[UPD] Update account_credit_control_dunning_fees.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-account_credit_control_dunning_fees
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control_dunning_fees/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-account_credit_control_dunning_fees
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-account_credit_control_dunning_fees/

OCA-git-bot committed:
Merge PR #280 into 16.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update sale_financial_risk.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-sale_financial_risk/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: credit-control-16.0/credit-control-16.0-sale_financial_risk
Translate-URL: https://translation.odoo-community.org/projects/credit-control-16-0/credit-control-16-0-sale_financial_risk/
